util/dvsim/FlowCfg.py

```diff
@@ -99,11 +99,11 @@ def __post_init__(self):
                 log.error("Parse error!\n%s", self.cfgs)
                 sys.exit(1)
 
-    @staticmethod
```

Comment:
Thanks for this change. Smushing different unrelated updates into the same PR makes the review hard. Would you please split this out to a separate PR?

Reply:
Oh, good point. The original version of this PR used this (because it passed a `dv_root` parameter around), but it's not needed since @weicaiyang's suggestion. I'll split this out now.